csharp-basics/exercises/Arithmetic/Exercise8/Program.cs

```diff
@@ -0,0 +1,46 @@
+﻿using System;
+
+namespace Exercise8
+{
+    internal class Program
+    {
+        const int normalHours = 40;
+        const int maxHours = 60;
+        const int minWage = 8;
+        
+        public static void Main(string[] args)
+        {
+            WageCalc(7.5, 35);
+            WageCalc(8.2, 47);
+            WageCalc(10, 73);
+        }
+        
+        static void WageCalc(double basePay, int hoursWorked)
+        {
+            var overtimePay = 1.5 * basePay;
+            double wage;
```

Comment:
Neizskatās, ka uzdevuma nosacījumi ir izpildīti 
![image](https://user-images.githubusercontent.com/53189160/181070948-c4736363-4101-4430-83f9-2a2c1fc65912.png)

Reply:
Man ir šāds rezultāts
![image](https://user-images.githubusercontent.com/106081841/181096109-a017b1c4-726d-46a3-a813-5b29519c78ee.png)

Vienīgi skaitļi varēja būt ar 2 cipariem aiz komata, to izlabošu.